frame/elections-phragmen/src/lib.rs

```diff
@@ -479,10 +479,12 @@ pub mod pallet {
 		///   next round.
 		///
 		/// The dispatch origin of this call must be signed, and have one of the above roles.
-		///
-		/// # <weight>
 		/// The type of renouncing must be provided as witness data.
-		/// # </weight>
+		///
+		/// # Complexity
+		/// 	- Renouncing::Candidate(count): O(count + log(count))
+		/// 	- Renouncing::Member: O(1)
+		/// 	- Renouncing::RunnerUp: O(1)
```

Comment:
CI is failing on these lines, maybe put the `Renouncing::*` variants in backticks?

Reply:
Yes checked that, fixing it.